sunwell.js

```diff
@@ -95,6 +100,8 @@
     sunwell.settings.smallTextureFolder = sunwell.settings.smallTextureFolder || null;
     sunwell.settings.autoInit = sunwell.settings.autoInit || true;
     sunwell.settings.idAsTexture = sunwell.settings.idAsTexture || false;
+    sunwell.settings.misingImg = sunwell.settings.misingImg || null;
```

Comment:
I see a problem here: the missingImg can be a image and/or canvas, BUT - we cannot know if its already loaded when getMissingImg() is called, but it _needs_ to be loaded then.

Wouldn't it be better to pass an URL here and load it over the assetloader, since it also can load resources by URL directly?